apps/web/src/components/FormImageCard.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  Box,
+  Text,
+  Avatar,
+  AvatarGroup,
+  Tooltip,
+  flexbox,
+  Flex,
+} from '@chakra-ui/react';
+import { FormInstanceEntity, SignatureEntity } from './../../../web/src/client';
+import { useRouter } from 'next/router';
+import React from 'react';
+import { useStorage } from '@web/hooks/useStorage';
+import { Document, Page, pdfjs } from 'react-pdf';
+
+/**
+ * @param formName - the name of the form
+ * @param signatures - the signatures on the form
+ * @param link - the link to the form
+ * @returns a card for a form
+ */
+export const FormImageCard = ({
+  formInstance,
+  link,
+}: {
+  formInstance: FormInstanceEntity;
+  link: string;
+}) => {
+  const router = useRouter();
+  const { formURL } = useStorage(formInstance);
+
+  pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
+
+  let daysBetween = (date1: string, date2: string) => {
+    if (date1) {
+      const d1 = new Date(date1);
+      const d2 = new Date(date2);
+      const diffInTime = d2.getTime() - d1.getTime();
+      const diffInDays = Math.ceil(diffInTime / (1000 * 60 * 60 * 24));
+      return diffInDays;
+    }
+    return 0;
+  };
+
+  return (
+    <>
+      <Box
+        minW="246px"
+        minH="120px"
+        borderRadius="5px"
+        backgroundColor="#FFFFFF"
+        boxShadow="0px 0.5px 3px 1px #DCDCDC"
+        background="#FCFCFC"
+        cursor="pointer"
+        onClick={() => {
+          router.push(link);
+        }}
+        _hover={{
+          boxShadow: '0px 0.5px 6px 1px #DCDCDC',
+        }}
+      >
+        <Box>
```

Comment:
Do we need this `<Box>` tag? Doesn't seem like its doing anything.

Reply:
Could I also remove the <> </> tag in the beginning

Reply:
yep seems like it